standard_reports/item-usage/latest/convert_data_js/src/test/test.ts

```diff
@@ -0,0 +1,11 @@
+import { convert_data } from '../convert_data';
+import inputData from './input.json';
+import outputData from './output.json';
+import { describe, expect, it } from '@jest/globals';
+
+describe('Test all', () => {
```

Comment:
I think we need more 'overall' test for reports, rather then testing individual methods (unless overall fail and you need to drill down on the error)

Reply:
Hm, not sure I agree completely, I think both have value.

Little unit tests are much easier to write in test driven development, add a new test for each new piece you want to add. They are also much easier to ensure you have tested every case, different kinds of inputs to each individual method.

Overall tests are definitely crucial, like you say with real-looking data, to ensure it hangs together correctly. But I think this is a happy path test, does the correct report data come back with these one or two key use-case inputs.

Don't think we should be deleting the unit tests though, much easier to find what broke, and have confidence that each piece of the logic holds up in all the ways it should 👀

Reply:
You'll see comment below, and refactor. Which I wasn't 100% happy with, as it is a bigger change then what I intended in this PR, however I decided to make the change, since this is an example and we should try to keep examples simple, direct and as patterns that will be copied.